lib/ansible/modules/utilities/logic/include_vars.py

```diff
@@ -18,9 +18,11 @@
 module: include_vars
 short_description: Load variables from files, dynamically within a task
 description:
-  - Loads variables from a YAML/JSON files dynamically from within a file or from a directory recursively during task
-    runtime. If loading a directory, the files are sorted alphabetically before being loaded.
+  - Loads YAML/JSON variables dynamically from a file or from all files in a directory, recursively, during task runtime.
```

Comment:
why remove 'variables from a' ? that seems the important part of the phrase.

'all files' is not always true, as files are filtered by extension and/or the ignore option.

Reply:
I'll fix the "all files" bit. 
The old sentence had more `from`s than I thought were necessary - "from a YAML/JSON files . .  from within a file or from a directory". The new sentence is cleaner. But if it's not as clear, let's find a different way to say it.